ec2-spot-emr/emr-spot/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM maven:3.6.3-jdk-8
```

Comment:
The folder structure does not make much sense, right ?  I think is quite confusing to have a folder within a folder with two examples that are fundamentally different. 

I reckon we can leave the `emr-spot` as the entry directory, then move the previous content to a folder and the code within this PR to a different folder. Perhaps name the examples on the back of the technologies used. So basically `java_sdk_emr`, `python_sdk_emr`.  This will be better for navigation, IMO.